instrumentation-genai/opentelemetry-instrumentation-openai-v2/tests/test_chat_completions.py

```diff
@@ -659,6 +659,48 @@ def test_chat_completion_multiple_tools_streaming_no_content(
     )
 
 
+@pytest.mark.vcr()
+def test_chat_completion_with_content_span_sampled_out(
```

Comment:
again, close out if "sampled out" is common

```suggestion
def test_chat_completion_with_content_span_unsampled(
```